plugin/src/helpers/utils.ts

```diff
@@ -174,3 +174,14 @@ export const findModuleFromBase = ({ paths, candidates }): string | null => {
   }
   return null
 }
+
+export const isNextAuthInstalled = (): boolean => {
+  try {
+    // eslint-disable-next-line import/no-unassigned-import, node/no-extraneous-require, import/no-extraneous-dependencies
+    require('next-auth')
```

Comment:
This approach follows what's suggested in the npm documentation around [optional dependencies](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies)